jbpm-workitems/jbpm-workitems-core/src/main/java/org/jbpm/process/workitem/core/AbstractLogOrThrowWorkItemHandler.java

```diff
@@ -37,7 +37,7 @@
     protected String handlingProcessId;
     protected String handlingStrategy;
 
-    protected int retries = 3;
+    protected int retries = Integer.parseInt(System.getProperty("org.jbpm.exception.handlingStrategy.retry.count", "3"));
```

Comment:
Just a small suggestion. Most (if not all) of our system properties are not using camel case, so I would use something like 
```org.jbpm.exception.handling_strategy.retry.count```
But it's just a suggestion

Reply:
Updated system property as per suggestion.